source/print.d

```diff
@@ -84,21 +101,52 @@ class Printer {
 	    return render;
     }
 
+    /**
+     * Formats a list for display
+     * without table formatting.
+     *
+     * @param list          The List to be formatted.
+     */
+    string[] printListWithoutTable(List list) {
+      string[] render;
+      render ~= list.name;
+      render ~= getRowOuter(listWidth);
+      foreach (task; list.tasks) {
+        render ~= format("%s: %s", task.humanId, task.name); // TODO: implement listModePreserveWidth checking
+      }                                                      // possibly look at D string formatting functionality
+      render ~= " ";
+
+      return render;
+    }
+
     string[] printLists(List[] lists) {
+      bool isList = (0 == cmp(this.displayMode, "list"));
+
     	int size = 0;
     	foreach (list; lists) {
     		if (list.tasks.length > size)
     			size = to!int(list.tasks.length);
     	}
 
     	string[] print;
-    	foreach (list; lists) {
-    		string[] rows = printList(list, size);
-    		if (print.length == 0)
+
+    	foreach (x, list; lists) {
+        string[] rows;
+        if (isList) {
```

Comment:
This could be simplified to a ternary operator (again, very glad that this is not C):
`string[] rows = isList ? printList(list) : printList(list, size)`

Reply:
i get to use a TERNARY OPERATOR! my favorite!